tests/data_sources/test_cancer_hotspots.py

```diff
@@ -2,12 +2,14 @@
 
 import pytest
 from evidence.data_sources import CancerHotspots
+from tests.conftest import DATA_DIR
```

Comment:
this will fail in some testing environments -- you will probably want to either make an empty `__init__.py` file or define it as a fixture